MonoGame.Framework/Input/GamePadThumbSticks.cs

```diff
@@ -42,10 +42,47 @@ public Vector2 Right
             get { return _right; }
         }
 
+        public GamePadThumbSticks(Vector2 leftPosition, Vector2 rightPosition, float leftDeadZoneAmount, float rightDeadZoneAmount)
+            : this(leftPosition, rightPosition, GamePadDeadZone.None, GamePadDeadZone.None, leftDeadZoneAmount, rightDeadZoneAmount)
+        {
+            
+        }
+
         public GamePadThumbSticks(Vector2 leftPosition, Vector2 rightPosition)
             : this(leftPosition, rightPosition, GamePadDeadZone.None, GamePadDeadZone.None)
         {
-            
+
+        }
+
+        internal GamePadThumbSticks(Vector2 leftPosition, Vector2 rightPosition, GamePadDeadZone leftDeadZoneMode, GamePadDeadZone rightDeadZoneMode, float leftDeadZone, float rightDeadZone) : this()

```

Comment:
This should be public and documented

Reply:
The parameter name doesn't match the usage below. I prefer `xDeadZone` or `xDeadZoneSize` to `xDeadZoneAmount`.